packages/create-vite/README.md

```diff
@@ -53,6 +53,8 @@ Currently supported template presets include:
 - `preact-ts`
 - `lit`
 - `lit-ts`
+- `fast`
+- `fast-ts`
```

Comment:
I dropped these in as a logical grouping. I assume there is a method to the list here, so I refrained from sorting it alphabetically, etc. If that's something ya'll would prefer, let me know and I'm happy to update accordingly.